platforms/Windows/shared/swift.en-us.wxl

```diff
@@ -8,6 +8,7 @@
   <String Id="CliAsserts_ProductName" Value="Swift Command Line Tools (Asserts)" />
   <String Id="Dbg_ProductName" Value="Swift Debugging Tools" />
   <String Id="DbgAsserts_ProductName" Value="Swift Debugging Tools (Asserts)" />
+  <String Id="EmbeddedPython_ProductName" Value="Embedded Python 3.10.1" />
```

Comment:
Since this is variant specific msi, we should have the name reflect that.

```suggestion
  <String Id="EmbeddedPythonAsserts_ProductName" Value="Embedded Python 3.10.1 (Asserts)" />
```